src/libs/ui/src/templates/LandingPage/landingPage.styles.scss

```diff
@@ -1,123 +1,127 @@
 .test {
     color: red;
-}
+  }
```

Comment:
Please remove component css changes here , this styles sheet should have only landing page css

- [ ] remove Css classes here and add corresponding component